src/kv/deserialise.h

```diff
@@ -287,8 +288,9 @@ namespace kv
         return ApplyResult::FAIL;
       }
 
+      auto config = consensus->get_latest_configuration_unsafe();
       bool result = true;
-      auto r = history->verify_and_sign(sig, &term);
+      auto r = history->verify_and_sign(sig, &term, config);
```

Comment:
I think this could be better done as a consensus hook, and would require fewer state pointers.

Reply:
#2741